modules/node/src/admin/admin.provider.ts

```diff
@@ -129,6 +133,11 @@ class AdminMessaging extends AbstractMessagingProvider {
       "admin.get-channels-for-merging",
       this.authService.useAdminToken(this.getChannelsForMerging.bind(this)),
     );
+
+    await super.connectRequestReponse(
+      "admin.fix-proxy-addresses",
+      this.authService.useAdminToken(this.fixProxyFactoryAddresses.bind(this)),
+    );
```

Comment:
my guess is well want to do a dry run here? or at least also have access to the `get-bad-proxy` addresses functionality